src/inventory/internal/InventoryRenderer.gd

```diff
@@ -3,6 +3,12 @@ extends Control
 ## Renders a given inventory and handles basic input events - dragging and dropping items
 ## from and to itself and selecting items.
 
+## Constant ID for the selected item.
+const ID_SELECTION: String = "SELECTION"
+
```

Comment:
Selection shouldn't be handled by the renderer, only hovering. It should be up to other things to implement their own selection.